tests/unit/BlockStorageMultiTx.d

```diff
@@ -72,6 +74,42 @@ private void main ()
     size_t idx;
 
     assert(loaded_blocks == blocks);
+
+    storage.release();
+    checksumTest(path);
+}
+
+private void checksumTest (string path)
+{
+    BlockStorage storage = new BlockStorage(path);
+
+    // Checksum test
+    auto data = "agora".representation;
+    immutable ubyte[4] expected = cast(ubyte[4]) [0xfd, 0xa0, 0xb6, 0xa0];
+    const ubyte[] result = storage.checksum(data);
+    assert(result == expected);
+
+    // Noise Test
+    File block_file;
+    string file_name = buildPath(path, storage.getFileName(0));
+    block_file = File(file_name, "r+b");
+    block_file.seek(5, SEEK_SET);
+    ubyte[1] noise;
+    foreach (i;0 .. noise.length)
+        noise[i] = cast(ubyte)(0);
```

Comment:
What was the intention here? Because this doesn't do anything. `ubyte[1]` is already a fixed-length array with all values set to zero.

Reply:
Yes, It's 0 So, I will changed the value to 9